hydra-node/src/Hydra/Chain/Direct/Wallet.hs

```diff
@@ -311,9 +312,7 @@ coverFee_ pparams systemStart epochInfo lookupUTxO walletUTxO partialTx@Babbage.
         unbalancedTx
           & bodyTxL . outputsTxBodyL %~ (|> feeTxOut)
           & bodyTxL . feeTxBodyL .~ Coin 10_000_000
-      -- XXX: Not hard-code but parameterize to make this flexible enough for
-      -- later signing and commit transactions with more than one sig
-      additionalWitnesses = 2
+      additionalWitnesses = 1 + length (partialTx ^. bodyTxL . reqSignerHashesTxBodyL)
```

Comment:
Should: Explain why the `1` is needed still.

I _think_ it's because it would account for the fee input we selected above, but not entirely sure - are you?